framework/encryption/__init__.py

```diff
@@ -6,13 +6,22 @@
 
 SENSITIVE_DATA_KEY = jwe.kdf(settings.SENSITIVE_DATA_SECRET.encode('utf-8'), settings.SENSITIVE_DATA_SALT.encode('utf-8'))
 
+def ensure_str(value):
```

Comment:
What I think @chrisseto is saying is that he's going to upgrade the osf to python 3 while he's in GA.